Pipfile

```diff
@@ -23,6 +23,9 @@ flake8 = "*"
 python-multipart = "*"
 sqlalchemy = "*"
 uvicorn = "*"
+bcrypt = "*"
+PyJWT = "*"
+passlib = "*"
```

Comment:
Did you mean to include these?

Reply:
when installing requirements via pip, those were missing and had to install manually in order to run the tests. i was waiting feedback whether that's correct or just my issue.

Reply:
They are only dev requirements, which is why I think they weren't included.

Reply:
reverted this change. i suppose if nobody needed them so far, they were only issue because of the pip install i did

Reply:
Yeah, I'm not completely happy with Pipenv. It's OK like this for now.

I might switch the environment handling to pure `python -m venv`, Poetry or something. But that will be later.